services/QuillLMS/db/big_query/views/reporting_sessions_view.sql

```diff
@@ -1,31 +1,28 @@
 SELECT
- activity_sessions.user_id as student_id,
- activity_sessions.activity_id as activity_id,
- activities.name as activity_name,
- schools.id as school_id,
- schools.name as school_name,
- classrooms_teachers.user_id as teacher_id,
- users.name as teacher_name,
- classrooms.id as classroom_id,
- classrooms.name AS classroom_name,
- classrooms.grade as grade,
- DATETIME_TRUNC(activity_sessions.completed_at, DAY) as completed_date,
- SUM(activity_sessions.timespent) as time_spent,
- SUM(activities.question_count) as question_count,
- Count(activity_sessions.id) as activity_count
+  activity_sessions.id as session_id,
```

Comment:
Note, I ran into some double counting data issues doing the sums of counts, so I decided to make this table grouped by activity_session and took away the aggregate counts.